Library/Homebrew/cask/cask_loader.rb

```diff
@@ -244,7 +244,7 @@ def self.can_load?(ref)
 
       def initialize(token, from_json: nil)
         @token = token.sub(%r{^homebrew/(?:homebrew-)?cask/}i, "")
-        @path = CaskLoader.default_path(token)
+        @path = CaskLoader.default_path(@token)
```

Comment:
Fixes an issue where args in the form "user/repo/name" wouldn't be handled properly.